### PR TITLE
Devise views

### PR DESCRIPTION
**Pull request by tochman**

### CODE REVIEW

File: features/approve_and_assign_users.feature [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @javascript
-Feature: As a system adminidtrator
+Feature: User administration
+  As a system adminidtrator
```
Comment (diraulo) [inline features/approve_and_assign_users.feature]: not a blocker but you have a typo here 😉

File: features/step_definitions/steps.rb [hunk 2]
```diff
@@ -124,7 +124,8 @@
 end
 
 Then(/^show me an image of the page$/) do
-  sleep(0.5) until page.evaluate_script('$.active') == 0
+  #sleep(0.5) until page.evaluate_script('$.active') == 0
```
Comment (diraulo) [inline features/step_definitions/steps.rb]: Perhaps we can remove this line altogether?

File: features/step_definitions/steps.rb [hunk 3]
```diff
@@ -159,7 +162,9 @@
 end
 
 And(/^I click "([^"]*)"$/) do |value|
-  find_button(value).trigger('click')
+  within 'main' do
```
Comment (diraulo) [inline features/step_definitions/steps.rb:164]: How's this step definition different from the one at line 184

File: config/puma.rb [hunk 4]
```diff
@@ -1,11 +1,14 @@
 threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i
 threads threads_count, threads_count
 
-port        ENV.fetch("PORT") { 3000 }
-environment ENV.fetch("RAILS_ENV") { 'production' }
+port ENV.fetch('PORT') { 3000 }
+environment ENV.fetch('RAILS_ENV') { 'production' }
+
+if Rails.env.production?
```
Comment (diraulo) [inline config/puma.rb:7]: 👍

**Review by diraulo — COMMENTED** [review]
Few comments for you @tochman

File: features/step_definitions/steps.rb (hunk 2 above)
Comment (tochman) [reply features/step_definitions/steps.rb]: Yes

File: features/step_definitions/steps.rb (hunk 3 above)
Comment (tochman) [reply features/step_definitions/steps.rb:164]: I had some issues with Poltergeist being able to interact with the links/buttons using Capybara so I created 2 step defs. A bit hacky I know

**Review by diraulo — APPROVED** [review]

**Pull request merged**